### PR TITLE
fix(install): Return Code 0 if venv already exists

### PR DESCRIPTION
**Pull request by wombelix**
* [ ] I have added an entry to `docs/changelog.md`

## Summary of changes
Related Issue: https://github.com/pypa/pipx/issues/125

Restores functionality introduced in Commit: https://github.com/pypa/pipx/commit/11b853e9c6926b32133b27822516b2a5b4f35411

## Test plan

Tested by running and checking the return code
```
pipx install <alreadyinstalledpackage>
echo $?
# expected result: 0
```

### CODE REVIEW

**Review by gaborbernat — CHANGES_REQUESTED** [review]
You'll need to validate with test, add changelog.

**Comment by wombelix** [comment]
> You'll need to validate with test, add changelog.

Thanks @gaborbernat and apologize for not taking enough attention to the contributing guide earlier. I pushed a new commit that address the failed test, local run was now successful, also added an changelog entry.

File: docs/changelog.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 dev
 
 - Fixed `pipx list` output phrasing to convey that python version displayed is the one with which package was installed. 
+- Fixed `pipx install` to provide return code 0 if venv already exists, similar behavior as pip. (#736)
```
Comment (uranusjr) [inline docs/changelog.md:4]:
```suggestion
- Fixed `pipx install` to provide return code 0 if venv already exists, similarly to pip’s behavior. (#736)
```

Comment (wombelix) [reply docs/changelog.md]: Thanks for the suggestion, updated and new commit pushed accordingly.

**Review by uranusjr — APPROVED** [review]

**Review by gaborbernat — APPROVED** [review]

**Pull request merged**